pkg/cmd/cli/cmd/deploy.go

```diff
@@ -26,6 +26,12 @@ NOTE: This command is still under active development and is subject to change.`
 	deploy_example = `  // Display the latest deployment for the 'database' deployment config
   $ %[1]s deploy database
 
+  // List all deployments for the 'database' deployment config
+  $ %[1]s deploy database -L
```

Comment:
Describe could give you that, for sure. 

Still, I do like this idea of having it somehow attached to the `deploy` command, because user-friendliness (the flow of doing `osc deploy database --latest` then `osc deploy database -L` seemed fluid). Also, displaying the complete deployment history in describe could become too much depending on the history size, and given describe doesn't provide an argument to control it (limit the history size, etc).

We could do a mid-term, where we add a few more items from the history to describe, but still limit to say 3 or 5, and add more information to this one, similar to what describe does (maybe just a little less verbose, say not displaying the pod status and things like that). How does it sound?